libs/login/build.gradle

```diff
@@ -36,68 +23,40 @@ android {
     }
 }
 
-ext {
-    // libs
-    wordpressLintVersion = '2.1.0'
-    wordpressUtilsVersion = '3.5.0'
-    gravatarSdkVersion = '0.2.0'
-
-    // main
-    androidxAppCompatVersion = '1.7.0'
-    androidxCoreVersion = '1.12.0'
-    androidxConstraintLayoutVersion = '2.1.4'
-    daggerVersion = '2.52'
-    glideVersion = '4.16.0'
-    javaxAnnotationVersion = '10.0-b28'
-    materialVersion = '1.12.0'
-    playServicesAuthVersion = '18.1.0'
-    credentialManagerVersion = '1.2.0'
-
-    // test
-    androidxArchCoreVersion = '2.1.0'
-    assertjVersion = '3.26.3'
-    junitVersion = '4.13.2'
-    mockitoVersion = '2.28.2'
-    robolectricVersion = '4.9'
-}
-
 dependencies {
-    implementation ("org.wordpress:utils:$wordpressUtilsVersion") {
-        exclude group: "com.android.volley"
-    }
-    implementation ("com.gravatar:gravatar:$gravatarSdkVersion")
+    implementation libs.wordpress.utils
+    implementation libs.gravatar
 
-    implementation "androidx.appcompat:appcompat:$androidxAppCompatVersion"
+    implementation libs.androidx.appcompat.main
 
-    implementation "androidx.constraintlayout:constraintlayout:$androidxConstraintLayoutVersion"
-    implementation "com.google.android.material:material:$materialVersion"
+    implementation libs.androidx.constraintlayout.main
+    implementation libs.google.material
 
-    implementation "androidx.core:core:$androidxCoreVersion"
+    implementation libs.androidx.core.main
 
-    api "com.google.android.gms:play-services-auth:$playServicesAuthVersion"
+    api libs.google.play.services.auth
 
     implementation project(":libs:fluxc")
 
-    implementation "com.github.bumptech.glide:glide:$glideVersion"
-    kapt "com.github.bumptech.glide:compiler:$glideVersion"
+    implementation libs.bumptech.glide.main
+    kapt libs.bumptech.glide.compiler
 
-    implementation "androidx.credentials:credentials:$credentialManagerVersion"
-    implementation "androidx.credentials:credentials-play-services-auth:$credentialManagerVersion"
+    implementation libs.androidx.credentials
+    implementation libs.androidx.credentials.play.service.auth
 
     // Dagger
-    implementation "com.google.dagger:dagger:$daggerVersion"
-    kapt "com.google.dagger:dagger-compiler:$daggerVersion"
-    compileOnly "org.glassfish:javax.annotation:$javaxAnnotationVersion"
```

Comment:
This dependency was removed intentionally as it doesn't seem it's unsed anywhere. The compilation was successful without it.